parts/windowsProvision.ps1

```diff
@@ -20,28 +262,34 @@ function InstallOpenSSH()
             $list = (Get-WindowsCapability -Online | ? Name -like 'OpenSSH.Server*')
             if ($list) {
                 Add-WindowsCapability -Online -Name $list.Name
+                Install-PackageProvider -Name "NuGet" -Force
                 Install-Module -Force OpenSSHUtils
             } else {
-                $open_ssh_uri = "https://github.com/PowerShell/Win32-OpenSSH/releases/download/v7.7.2.0p1-Beta/OpenSSH-Win64.zip"
-                $open_ssh_file = "C:/tmp/OpenSSH-Win64.zip"
-                & curl.exe -L -o $open_ssh_file $open_ssh_uri
-                & 7z x $open_ssh_file -oC:/tmp
-                c:/tmp/OpenSSH-Win64/install-sshd.ps1
+                $installDir = Join-Path $PACKAGES_DIRECTORY "OpenSSH"
+                Install-ZipCITool -ZipPath $PACKAGES["dig"]["local_file"] `
+                                  -InstallDirectory $installDir 
+                $installDir/OpenSSH-Win64/install-sshd.ps1
             }
         }
+
         Start-Service sshd
         & netsh advfirewall firewall add rule name="SSH TCP Port 22" dir=in action=allow protocol=TCP localport=22

```

Comment:
Instead of this `netsh` call and the check for `$LASTEXITCODE`, we could use native PowerShell cmdlets generating exceptions on errors.

This is the PowerShell cmdlet usage for the same firewall rule to be opened:
```
New-NetFirewallRule -Name "ssh-tcp-rule" -DisplayName "SSH TCP Port 22" -LocalPort 22 -Action Allow -Enabled True -Direction Inbound -Protocol TCP -Profile Any
```